### ISSUE TITLE
LDAP/SAML : LDAP users not validated in the directory after logged in via SAML

### ISSUE THREAD

**Issue by brenard** [nextcloud/server#29006]
### Steps to reproduce
1. Configure user_saml app with _require_provisioned_account_ enabled
2. Configure user_ldap app with a LDAP user filter like _(&(objectClass=person)(myAllowedServices=nextcloud))_
3. Create a enabled user in LDAP (use by your SSO as users backend) **with** Nextcloud access
4. Log in on Nextcloud using SSO : it's supposed to work
5. Log out on Nextcloud and SSO
6. Remove Nextcloud access in LDAP
7. Try to log in on Nextcloud using SSO : it's supposed to not work with  but you still can access Nextcloud

### Expected behaviour

On step 7, user should not be able to access Nextcloud with an error message saying that your account is not provisioned (triggered by _user_saml_ app).

### Actual behaviour

On step 7, access normally to Nextcloud.

**Observation:** Nextcloud does not search for the user in the LDAP directory during connection (according to LDAP logs).

### Server configuration

**Operating system:** Debian Buster (10.10)

**Web server:** Apache 2.4.38

**Database:** PosgreSQL 11.12

**PHP version:** 7.3.29

**Nextcloud version:** 22.1.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated many time from older many older nextcloud & ownCloud versions.

**Where did you install Nextcloud from:** Dedicated server (with dedicated server for the database)

**Signing status:**
<details>
<summary>Signing status</summary>

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- user_saml
	- EXCEPTION
		- OC\IntegrityCheck\Exceptions\InvalidSignatureException
		- Signature data not found.

Raw output
==========
Array
(
    [user_saml] => Array
        (
            [EXCEPTION] => Array
                (
                    [class] => OC\IntegrityCheck\Exceptions\InvalidSignatureException
                    [message] => Signature data not found.
                )

        )

)


```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.7.0
  - activity: 2.15.0
  - admin_audit: 1.11.0
  - circles: 22.1.1
  - cloud_federation_api: 1.4.0
  - comments: 1.11.0
  - contactsinteraction: 1.2.0
  - dashboard: 7.1.0
  - dav: 1.18.0
  - federatedfilesharing: 1.11.0
  - federation: 1.11.0
  - files: 1.16.0
  - files_mindmap: 0.0.25
  - files_pdfviewer: 2.3.0
  - files_sharing: 1.13.2
  - files_trashbin: 1.11.0
  - files_versions: 1.14.0
  - files_videoplayer: 1.11.0
  - firstrunwizard: 2.11.0
  - impersonate: 1.9.0
  - logreader: 2.7.0
  - lookup_server_connector: 1.9.0
  - nextcloud_announcements: 1.11.0
  - notifications: 2.10.1
  - oauth2: 1.9.0
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.11.0
  - richdocuments: 4.2.3
  - serverinfo: 1.12.0
  - settings: 1.3.0
  - sharebymail: 1.11.0
  - survey_client: 1.10.0
  - systemtags: 1.11.0
  - text: 3.3.0
  - theming: 1.12.0
  - twofactor_backupcodes: 1.10.1
  - updatenotification: 1.11.0
  - user_ldap: 1.11.0
  - user_saml: 4.1.1
  - user_status: 1.1.1
  - viewer: 1.6.0
  - weather_status: 1.1.0
  - workflowengine: 2.3.1
Disabled:
  - encryption
  - files_external
  - files_rightclick
  - recommendations
  - support
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "22.1.1.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "ldapIgnoreNamingRules": false,
        "theme": "",
        "maintenance": false,
        "loglevel": "2",
        "logfile": "\/var\/log\/nextcloud\/nextcloud.log",
        "logtimezone": "Europe\/Paris",
        "trusted_domains": [
            "owncloud.enercoop.org",
            "clood.enercoop.org"
        ],
        "share_folder": "\/",
        "mail_smtpmode": "smtp",
        "filesystem_check_changes": 0,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "activity_expire_days": 30,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0
        },
        "asset-pipeline.enabled": true,
        "trashbin_retention_obligation": "7,14",
        "versions_retention_obligation": "auto,365",
        "appstore.experimental.enabled": false,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "updater.release.channel": "stable",
        "overwrite.cli.url": "https:\/\/clood.enercoop.org",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "default_language": "fr",
        "data-fingerprint": "b4048ae237bdee6bbf0c497e69e021aa",
        "ldapUserCleanupInterval": "10",
        "mail_sendmailmode": "smtp",
        "defaultapp": "files",
        "default_phone_region": "FR"
    }
}
```
</details>

**Are you using external storage, if yes which one:** local only

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
+-------------------------------+----------------------------------------------------------------+
| Configuration                 |                                                                |
+-------------------------------+----------------------------------------------------------------+
| hasMemberOfFilterSupport      | 1                                                              |
| homeFolderNamingRule          |                                                                |
| lastJpegPhotoLookup           | 0                                                              |
| ldapAgentName                 | uid=nextcloud,ou=sysaccounts,o=enercoop                        |
| ldapAgentPassword             | ***                                                            |
| ldapAttributesForGroupSearch  |                                                                |
| ldapAttributesForUserSearch   |                                                                |
| ldapBackupHost                |                                                                |
| ldapBackupPort                |                                                                |
| ldapBase                      | o=enercoop                                                     |
| ldapBaseGroups                | o=enercoop                                                     |
| ldapBaseUsers                 | o=enercoop                                                     |
| ldapCacheTTL                  | 600                                                            |
| ldapConfigurationActive       | 1                                                              |
| ldapDefaultPPolicyDN          |                                                                |
| ldapDynamicGroupMemberURL     |                                                                |
| ldapEmailAttribute            | mail                                                           |
| ldapExperiencedAdmin          | 1                                                              |
| ldapExpertUUIDGroupAttr       |                                                                |
| ldapExpertUUIDUserAttr        | uid                                                            |
| ldapExpertUsernameAttr        | uid                                                            |
| ldapExtStorageHomeAttribute   |                                                                |
| ldapGidNumber                 | gidNumber                                                      |
| ldapGroupDisplayName          | cn                                                             |
| ldapGroupFilter               | (|(objectclass=eneGroup)(objectclass=eneDynGroup))             |
| ldapGroupFilterGroups         |                                                                |
| ldapGroupFilterMode           | 1                                                              |
| ldapGroupFilterObjectclass    | eneGroup                                                       |
| ldapGroupMemberAssocAttr      | uniqueMember                                                   |
| ldapHost                      | ldaps://ldap.enercoop.org                                      |
| ldapIgnoreNamingRules         |                                                                |
| ldapLoginFilter               | (&(objectclass=enepeople)(uid=%uid)(eneAllowedServices=clood)) |
| ldapLoginFilterAttributes     |                                                                |
| ldapLoginFilterEmail          | 0                                                              |
| ldapLoginFilterMode           | 1                                                              |
| ldapLoginFilterUsername       | 1                                                              |
| ldapMatchingRuleInChainState  | unknown                                                        |
| ldapNestedGroups              | 0                                                              |
| ldapOverrideMainServer        | 0                                                              |
| ldapPagingSize                | 500                                                            |
| ldapPort                      | 636                                                            |
| ldapQuotaAttribute            |                                                                |
| ldapQuotaDefault              |                                                                |
| ldapTLS                       | 0                                                              |
| ldapUserAvatarRule            | default                                                        |
| ldapUserDisplayName           | cn                                                             |
| ldapUserDisplayName2          |                                                                |
| ldapUserFilter                | (&(objectclass=enePeople)(eneAllowedServices=clood))           |
| ldapUserFilterGroups          |                                                                |
| ldapUserFilterMode            | 1                                                              |
| ldapUserFilterObjectclass     | enePeople                                                      |
| ldapUuidGroupAttribute        | auto                                                           |
| ldapUuidUserAttribute         | auto                                                           |
| turnOffCertCheck              | 0                                                              |
| turnOnPasswordChange          | 0                                                              |
| useMemberOfToDetectMembership | 1                                                              |
+-------------------------------+----------------------------------------------------------------+
```
</details>

### Client configuration
**Browser:** Firefox 92.0.1

**Operating system:** Debian Bulleyes

### Logs

No log seems convincing. Please say me witch type of logging could be usefull for you.

#### Web server error log
<details>
<summary>Web server error log</summary>

```
No error log entry during user connection.
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No error log entry during user connection.
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
No OC found index.js:46:12
JQMIGRATE: Migrate is installed, version 3.3.2 jquery-migrate.min.js:2:708
Cette page utilise la propriété non standard « zoom ». Envisagez d’utiliser calc() dans les valeurs des propriétés pertinentes ou utilisez « transform » avec « transform-origin: 0 0 ». injectStylesIntoStyleTag.js:190:10
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:62:15
[DEBUG] files: Templates providers 
Object { 0: {…}, 1: {…}, 2: {…}, 3: {…}, app: "files", uid: "eetest.01" }
ConsoleLogger.js:42:18
[DEBUG] files: Templates folder 
Object { app: "files", uid: "eetest.01", templatesPath: "Templates/" }
ConsoleLogger.js:42:18
updating timezone and offset for SAML user timezone.js:24:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Uncaught ReferenceError: jstz is not defined
    <anonymous> https://clood.enercoop.org/apps/user_saml/js/timezone.js?v=6a1157d4-8:30
timezone.js:30:3
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Proxying an event bus of version 1.3.0 with 2.0.0 ProxyBus.ts:17:12
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities 8 capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 3 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
OCA.Files.Sidebar initialized Sidebar.js:36:10
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:62:15
OCA.Comments.View initialized comments-app.js:32:8
OCA.Sharing.ShareSearch initialized ShareSearch.js:33:10
OCA.Sharing.ExternalLinkActions initialized ExternalLinkActions.js:33:10
Proxying an event bus of version 1.3.0 with 2.0.0 ProxyBus.ts:17:12
OCA.Viewer initialized Viewer.js:49:10
Proxying an event bus of version 1.3.0 with 2.0.0 ProxyBus.ts:17:12
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
Proxying an event bus of version 1.3.0 with 2.0.0 ProxyBus.ts:17:12
Loading theming data for notification bell styling App.vue:138
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
Registering notifications container as a menu App.vue:148
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
Notifications permissions not yet requested App.vue:381
Polling interval updated to 30000 App.vue:325
Started background fetcher as session_keepalive is enabled App.vue:204
OCA.Files.Settings initialized Settings.js:29:10
Content Security Policy: Les paramètres de la page ont empêché le chargement d’une ressource à eval (« script-src »). global.js:10:10
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 96 globals.js:62:15
session heartbeat polling started session-heartbeat.js:101:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 15 globals.js:62:15
Please do NOT wait for the DOMContentLoaded before registering your viewer handler Viewer.vue:263
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 5 globals.js:62:15
Got notification data App.vue:288
Polling interval updated to 30000 App.vue:325
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 22 globals.js:62:15
Clipboard is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 18 globals.js:62:15
registerDetailView is deprecated! It will be removed in nextcloud 20. merged-index.js:7840:12
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
registerTabView is deprecated! It will be removed in nextcloud 20. merged-index.js:7814:12
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 32 globals.js:62:15
[DEBUG] unified-search: Unified Search initialized with the following providers 
Object { 0: {…}, 1: {…}, 2: {…}, 3: {…}, 4: {…}, 5: {…}, 6: {…}, 7: {…}, app: "unified-search", uid: "eetest.01" }
ConsoleLogger.js:42:18
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 16 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities 2 capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
OC.getCapabilities is deprecated and will be removed in Nextcloud 21. See @nextcloud/capabilities capabilities.js:35:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 8 globals.js:62:15
querying Whats New data was successful: nocontent whatsnew.js:56:9
undefined whatsnew.js:57:9
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 22 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 23 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 23 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 25 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 26 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 24 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 23 globals.js:62:15
OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:123:11
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment util.js:135:11
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 30 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 12 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 4 globals.js:62:15
$ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 2 globals.js:62:15
jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own. 148 globals.js:62:15
No new notification data received App.vue:293
Polling interval updated to 30000 App.vue:325
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment 7 util.js:135:11
No new notification data received App.vue:293
Polling interval updated to 30000 App.vue:325
OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment 7 util.js:135:11
```
</details>

**Comment by brenard** [comment]
I investigated on this bug and I found the explanation :

-  _user_saml_ calls the `userExists()` method of the different user backends to see if the user is well provisioned
- in the case of the _user_ldap_ backend, for the `userExists ()` method, if the user exists in the database, this is sufficient. Moreover, this `userExists()` method cache whether the user exists or not. As a result, the LDAP directory will not be consulted if the user has recently connected (or if an action by another user in the application to verify its existence).

I tried to quickly patch the method `userExists()` to confirm user exists in LDAP (using `userExistsOnLDAP()`) and it works but it does not solve the problems related to the cache. By disabling the cache in `userExists()` and ̀`userExistsOnLDAP()` methods, the application behaves as expected, but this seems to have a strong impact on the performance of Nextcloud (and potentially of the LDAP directory): I had to backtrack very quickly, because the load of the machine was exploding...

In short, I understand the problem better, but I do not know how to solve it immediately without it having too much impact.

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+
